doc/release-notes/11605-existing-shib-external-users-auth.md

```diff
@@ -0,0 +1,2 @@
+Implemented a new feature flag ``dataverse.feature.api-bearer-auth-use-shib-user-on-id-match``, which allows the use of
+a Shibboleth user account when an identity match is found during API bearer authentication.
```

Comment:
Just wordsmithing - feel free to change further: the flag 'supports use of the new Dataverse client in instances that have historically allowed login via Shibboleth. Specifically, with this flag, when an OIDC bridge is configured to allow OIDC login with validation by the bridged Shibboleth providers, users with existing Shibboleth-based accounts in Dataverse can login to those existing accounts and thereby maintain access to their existing content and retaining their roles. (For security reasons, Dataverse's current support for direct login via Shibboleth cannot be supported in browser-based clients.)'